package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "json-rpc-peer",
-  "version": "0.17.0",
+  "version": "0.17.1",
```

Comment:
Unnecessary, I can do it before publishing.

Reply:
Yes, if we are publishing the npm module manually then we can do it before publishing.

As we are planning to publish to the NPM automatically through GitHub Actions, I believe it will be better to have a specific version number for each commit, and that's the reason why I bump this patch version here.